Utilities/Dockers/Alphashape/generate_alphashape.py

```diff
@@ -11,14 +11,10 @@
 parser.add_argument("-a", "--alpha", default = 8.0)
 args = parser.parse_args()
 
-# Get training data
-# Expected input format:
-# Tab-delimited file with at least two columns where the first two columns are PC1 and PC2 (in this order)
-# SAMPLE_ID might be added as the first column later based on code review
-training_points = []
+# Read training data
+# Expected input format: tsv with at least two columns where the first two columns are PC1 and PC2
```

Comment:
doesn't need to be first two columns, since you reference them by name below.  just need a PC1 and PC2 column

Reply:
Done